### PR TITLE
Changed pivot bar color

### PR DESCRIPTION
**Pull request by aayush848484**
Changed the pivot bar color on the Maintenance page.

# Type of change? 

- [x] Bug fix
- [ ] New feature
- [ ] Enhancement
- [ ] Breaking change (breaks backward compatibility)

# Description, Context, Motivation 
The pivot-bar in the maintenance page did not have high contrast. So, I changed it to #FFFFFF for better contrast. 

Demo:
image

...

**Checklist:**

- [x] All tests passed
- [x] The code follows the code style and conventions of this project
- [ ] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: src/components/pages/maintenance/maintenance.scss [hunk 1]
```diff
@@ -78,7 +78,8 @@
 
         &.active {
           color: themed('colorContentText');
-          border-bottom: 4px solid themed('colorHeaderBorderColor');
+          border-bottom: 4px solid;
+          border-bottom-color:  #FFFFFF;
```
Comment (spryor) [inline src/components/pages/maintenance/maintenance.scss]: This isn't going to work. All colors need to come from the theme file. If you go to settings and change the page to the light theme, the border will not be visible anymore. Notice how in the previous code the color was defined using the `themed` mixin. The only change you need is to change the value `colorHeaderBorderColor` maps to in the theme file.

File: src/styles/_themes.scss [hunk 2]
```diff
@@ -228,6 +228,7 @@ $themes: (
     colorHeaderIconHover: $colorWhite,
     colorHeaderItemBgHover: $colorMidnight,
     colorHeaderBorderColor: #c8c8c8,
+    colorHeaderOnTapColor: #333333,
```
Comment (spryor) [inline src/styles/_themes.scss]: Why is this variable being added? It doesn't appear it is ever being used.

**Comment by aayush848484** [comment]
Thanks spryor for pointing that out. I hadn't realized that we had an option to change between dark and light themes in the web-ui and that it was being controlled from the themes file. So, I've made the necessary changes and it works for both the themes. 

Dark Theme:
image

Light Theme:
image

In addition, I accidentally committed .vscode files. So, I had re-commit to make sure that those files were removed from the remote branch. 

Lastly, please let me know if you have any comments or if I need to make any changes.

File: src/styles/_themes.scss [hunk 3]
```diff
@@ -88,7 +88,7 @@ $themes: (
     colorHeaderText: $colorSmoke,
     colorHeaderIconHover: $colorWhite,
     colorHeaderItemBgHover: $colorMidnight,
-    colorHeaderBorderColor: $colorLead,
+    colorHeaderBorderColor: #FFFFFF,
```
Comment (spryor) [inline src/styles/_themes.scss]: Whenever you have a color hex value in css that is a repetition of the same value, you can truncate it to a three character shorthand. e.g. `#333333` becomes `#333`. We always use the shorthand when possible ([see the code style guide](https://github.com/Azure/pcs-remote-monitoring-webui/wiki/Code-Guide#css)). We also lowercase all the hex values.

File: .vscode/launch.json [hunk 4]
```diff
@@ -1,16 +0,0 @@
-{
```
Comment (spryor) [inline .vscode/launch.json]: These files need to be re-added. If this branch is merged the it will remove these files.

**Comment by aayush848484** [comment]
Changed the hex codes according to the style guide and added the files back to the branch.

File: .vscode/arduino.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+    "board": "AZ3166:stm32f4:MXCHIP_AZ3166",
+    "configuration": "upload_method=OpenOCDMethod",
+    "port": "COM3"
+}
```
Comment (spryor) [inline .vscode/arduino.json]: Add new lines here. If you are in vscode the new lines at the end of the file will be added automatically.

File: .vscode/c_cpp_properties.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+                "limitSymbolsToIncludedHeaders": false,
+                "path": [
+                    "C:\\Users\\t-aagupt\\AppData\\Local\\Arduino15\\packages\\AZ3166\\hardware\\stm32f4\\1.3.5\\cores\\arduino"
+                ]
+            },
+            "intelliSenseMode": "msvc-x64"
+        }
+    ],
+    "version": 4
+}
```
Comment (spryor) [inline .vscode/c_cpp_properties.json]: newlines

File: .vscode/arduino.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
+{
+    "board": "AZ3166:stm32f4:MXCHIP_AZ3166",
```
Comment (spryor) [inline .vscode/arduino.json]: Is this an ID for a particular board?

Comment (aayush848484) [reply .vscode/arduino.json]: It seems that the .vscode file has contents about the other files that I edited through VSCode. I uploaded a script to the Azure IoT Board and this is the id of that board. So, do you want me to keep this file, or remove it?

**Comment by aayush848484** [comment]
Added white space at the end of the files.

File: .vscode/arduino.json (hunk 7 above)
Comment (spryor) [reply .vscode/arduino.json]: Seems like it should be removed.

Comment (aayush848484) [reply .vscode/arduino.json]: Yeah sure.

**Pull request merged**